nixos/modules/services/web-apps/nexus.nix

```diff
@@ -87,6 +87,10 @@ in
 
     users.groups."${cfg.group}" = {};
 
+    systemd.tmpfiles.rules = [
+      "d '${cfg.home}' - ${cfg.user} ${cfg.group} - -"
```

Comment:
Wouldn't [`createHome`](https://nixos.org/nixos/options.html#users.users.%3Cname%3F%3E.createhome) be better here?

Reply:
Yeah, that sounds good.